cli/pom.xml

```diff
@@ -99,6 +99,11 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.iotdb</groupId>
+            <artifactId>node-commons</artifactId>
+            <version>0.14.0-SNAPSHOT</version>
+        </dependency>
```

Comment:
need cli dependence commons-jar?  @qiaojialin @HTHou

Reply:
This PR is trying to fix importing timeseries like `root.sg.d1."a.b"`. `"a.b"` is a measurementId.
Use `org.apache.iotdb.commons.utils.PathUtils` to support it.